tools/rna_tools/nastiseq/nastiseq.xml

```diff
@@ -0,0 +1,175 @@
+<tool id="nastiseq" name="NASTIseq" version="1.0">
+  <description>Identify cis-NATs using ssRNA-seq</description>
+
+  <requirements>
+    <requirement type="package" version="1.0">r-nastiseq</requirement>
+  </requirements>
+  <stdio>
+        <regex match="Execution halted"
+           source="both"
+           level="fatal"
+           description="Execution halted." />
+        <regex match="Error in"
+           source="both"
+           level="fatal"
+           description="An undefined error occured, please check your intput carefully and contact your administrator." />
+  </stdio>
+  <command>
+<![CDATA[
+      Rscript '$script_file'
+]]>
+  </command>
+  <configfiles>
+    <configfile name="script_file">
+          library(NASTIseq)
+
+          genepos = read.table("${annotation}", header = TRUE, sep = "\t")
+          genepos\$attributes = as.character(genepos\$attributes)
+          rownames(genepos) = genepos\$attributes
+
+          pospairs = read.table("${positive_pair}", sep = "\t", as.is = TRUE)
+
+          smat = as.matrix(read.table("${count_smt}",  sep = "\t",  row.names = 1))
+          colnames(smat) = NULL
+
+          asmat = as.matrix(read.table("${count_asmt}",  sep = "\t",  row.names = 1))
+          colnames(asmat) = NULL
+
+          WRscore = getNASTIscore(smat, asmat)
+
+          negpairs = getnegativepairs(genepos)
+
+          WRpred = NASTIpredict(smat,asmat, pospairs, negpairs)
+
+          WRpred_rocr = prediction(WRpred\$predictions,WRpred\$labels)
+
+          thr = defineFDR(WRpred_rocr,0.05)
+
+          WR_names = FindNATs(WRscore, thr, pospairs, genepos)
+
+          write.table(WR_names\$newpairs, file = "output_newpairs.tsv", row.names = FALSE, col.names = FALSE, sep = "\t", quote = FALSE)
+
+          write.table(WR_names\$neworphan, file = "output_neworphan.tsv", row.names = FALSE, col.names = FALSE, sep = "\t", quote = FALSE)
+
+    </configfile>
+  </configfiles>
+  <inputs>
+    <param name="annotation" type="data" format="tabular" label="Annotation file"
```

Comment:
hold on, I am working on the GTF input. it will be done soon. forgot that the new commits would trigger the new travis testing.